projects/igniteui-angular/src/lib/services/csv/csv-exporter.ts

```diff
@@ -47,7 +47,7 @@ export class IgxCsvExporterService extends IgxBaseExporter {
     public onExportEnded = new EventEmitter<ICsvExportEndedEventArgs>();
 
     protected exportDataImplementation(data: any[], options: IgxCsvExporterOptions) {
-        const csvData = new CharSeparatedValueData(data, options.valueDelimiter);
+        const csvData = new CharSeparatedValueData(data, options.valueDelimiter, 'rowData');
```

Comment:
We might consider moving the rowData mapping here and remove the dataMemberPath argument.

Reply:
Agreed & Done.